### PR TITLE
Fix an error with the ViewResponseListener

### PR DESCRIPTION
**Pull request by Ener-Getick**
See #1301, https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1301#issuecomment-170088363, https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1171#discussion_r49223924

### CODE REVIEW

**Comment by Ener-Getick** [comment]
Since #1171, the ``ViewResponseListener`` do not longer extends the ``TemplateListener``. This was causing an issue because the ``TemplateListener`` converts ``_template`` to a ``TemplateReferenceInterface|string``.
This PR fixes this bug by alloying to use the ``TemplateListener`` with the ``ViewResponseListener`` and by doing the transformation internally when ``_view`` is used.

File: EventListener/ViewResponseListener.php [hunk 1]
```diff
@@ -149,7 +131,6 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
     public static function getSubscribedEvents()
     {
         return array(
-            KernelEvents::CONTROLLER => array('onKernelController', -100),
             KernelEvents::VIEW => 'onKernelView',
```
Comment (Ener-Getick) [inline EventListener/ViewResponseListener.php]: We may need to have a bigger priority, needs to be changed in another PR...

**Comment by kcivey** [comment]
I think the change in ViewResponseListener may now be required in 1.7 for people who upgrade SensioFrameworkExtraBundle from 3.0.12, because 3.0.13 is now guessing the template in onKernelController rather than onKernelView:
https://github.com/sensiolabs/SensioFrameworkExtraBundle/commit/82b8ad65409c99f693088ecbc09c857f01e3b3fa

**Comment by xabbuh** [comment]
@kcivey see #1355
